### PR TITLE
pj-139-rework-post-edit-view add js and rebuild a post-form

### PR DESCRIPTION
**Pull request by NykoEugen**
Rebuild post form for using creditor for editing post content. Add JS for display this form by button click. The edit form display when get and post. But have problem with saving editform. Have this error:
image

No idea how does it fix

### CODE REVIEW

**Comment by NykoEugen** [comment]
I thin this error is because ckeditor to add many tags (for example enstead of space add &nbsp;). Thats why we cant edit this form corectly

**Comment by NykoEugen** [comment]
I found my problem. The problem is in an instance=post. There are send all html cod ckeditor

**Comment by NykoEugen** [comment]
Fix it) Now edit post works on one page

File: posts/views.py [hunk 1]
```diff
@@ -128,6 +138,7 @@ def test_func(self) -> bool:
     def get(self, request: HttpRequest, post_slug: str) -> HttpResponse:
         post = get_object_or_404(Post, slug=post_slug)
         form = PostForm(instance=post)
+        print(form)
```
Comment (acman) [inline posts/views.py:141]: remove `print`.

File: posts/views.py [hunk 2]
```diff
@@ -110,6 +119,7 @@ def post(self, request: HttpRequest, post_slug: str) -> HttpResponse:
             {
                 "post_comment_form": post_comment_form,
                 "reply_comment_form": reply_comment_form,
+                # "edit_post_form": edit_post_form,
```
Comment (acman) [inline posts/views.py:122]: I think it could be removed

**Review by acman — CHANGES_REQUESTED** [review]
Looks fine, but there are a lot of forms there, so all cases with forms should be tested accurately. There are no guarantees that everything works as expected

**Comment by NykoEugen** [comment]
> Looks fine, but there are a lot of forms there, so all cases with forms should be tested accurately. There are no guarantees that everything works as expected

So how realize all this funktion in one view?

**Comment by acman** [comment]
I think it is ok to have all that forms in one view, just write tests for processings all cases and we will see if there are any conflicts

**Comment by NykoEugen** [comment]
Works fine but how to hide delete form, don't know. Tests in progress

**Review by acman — APPROVED** [review]
Looks really nice! I will try to test all forms

**Pull request merged**